pkg/pb/gc.proto

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package gc;
+
+
+import "gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+service GarbageCollection {
+    rpc Retain(RetainRequest) returns (RetainResponse);
+}
+
+message Filter {
+    uint32 seed = 1;
+    uint32 k = 2;
+    uint32 bits_per_element = 3;
+    bytes table = 4;
+}
+
+message RetainRequest {
+    google.protobuf.Timestamp creation_date = 1 [(gogoproto.stdtime) = true];
```

Comment:
I've just learnt this can be actually annotated with `[(gogoproto.stdtime) = true, (gogoproto.nullable) = false]`, which will make `timestamp` of type `time.Time` instead of `*time.Time`.